internal/hcsoci/network.go

```diff
@@ -6,6 +6,13 @@ import (
 )
 
 func createNetworkNamespace(coi *createOptionsInternal, resources *Resources) error {
+	op := "hcsoci::createNetworkNamespace"
+	log := logrus.WithField("coi.ID", coi.ID)
```

Comment:
`WithField(logfields.ContainerID, coi.ID)`

Reply:
Interesting. That's a different pattern to `GetNamespaceEndpoints` immediately below. Am I missing something?

Reply:
We dont know the container id below unfortunately. Until I do the work to forward the `context.Context` around we can't really log the appropriate identifiers. In this case though we know the container id so it should be used.

Reply:
Fixed